pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>org.ccci</groupId>
   <artifactId>red-egg</artifactId>
-  <version>5-SNAPSHOT</version>
+  <version>5</version>
```

Comment:
Does this mean you're going to change the version in give-ep-extensions to use 5 instead of 5-SNAPSHOT?